.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,53 @@
+<!-- HTML Comments can be left in place or removed. -->
+
+#### Proposed Changes ####
+
+<!-- Describe the big picture of your changes here to communicate to the maintainers why we should accept this pull request. -->
+
+<!-- Does this change require an update to documentation? -->
+
+#### Types of Changes ####
+
+<!-- What types of changes does your code introduce to RKE2? Bugfix, New Feature, Breaking Change, etc -->
+
+#### Verification ####
+
+<!-- How can the changes be verified? Please provide whatever additional information necessary to help verify the proposed changes. -->
```

Comment:
This looks like it's copied from the rke2 and k3s repos, but in this case I think we can remove since the types of changes are always "tests" and the verification is just "run the tests"

Reply:
done